### DISCUSSION TITLE
No matching distribution found for whisper

### DISCUSSION THREAD

**Discussion by baiwei0703** [openai/whisper#1007] [Q&A] [answered]
I tried to installing whisper by pip, when I installed autocut(https://github.com/mli/autocut.git). But ever have successed. I tried to changing env to python 3.7 or 3.9. Everytime it always alerts "ERROR: Could not find a version that satisfies the requirement whisper (unavailable) (from autocut) (from versions: 0.9.5, 0.9.6, 0.9.7, 0.9.8, 0.9.9, 0.9.10, 0.9.11, 0.9.12, 0.9.13, 0.9.14, 0.9.15, 0.9.16, 1.0.0, 1.0.1, 1.0.2, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.1.6, 1.1.7, 1.1.8, 1.1.9, 1.1.10)
ERROR: No matching distribution found for whisper (unavailable)". How can I do?

**Comment by jongwook** [comment, marked as answer]
It appears the version numbers are the [whisper](https://pypi.org/project/whisper/#history) package on PyPI that is an unrelated one from this project. Please try

```
pip install openai-whisper
```

or follow README.md to install Whisper using the latest commit.

**Reply by baiwei0703** [reply]
`pip install openai-whisper`   It works!!! 감사합니다.

**Reply by bluebluegrass** [reply]
I am getting the same error message. I have tried `pip install openai-whisper` but error persists. I am on python 3.9. Any suggestions? thank you!
